docs/_blog/teos-uses-ldk-to-build-open-source-watchtower.md

```diff
@@ -0,0 +1,33 @@
+---
+title: "TEOS uses LDK to build open-source watchtower"
+description: "Learn how TEOS built an open-source watchtower"
+date: "2023-01-14"
+authors:
+  - Sergi Delgado
+tags:
+  - Case Studies 
+--- 
+
+[The Eye of Satoshi (TEOS)](https://github.com/talaia-labs/rust-teos) s a bitcoin watchtower with a specific focus on lightning. Watchtowers are third-party viewing services that help prevent people from cheating on the lightning network by constantly monitoring the bitcoin blockchain for channel breaches. If found, a watchtower will react by penalizing the misbehaving party and sending all funds to its counterpart.
+
+The Eye of Satoshi (TEOS) needs information from several building blocks in order to work. This includes block data processing (including reacting to forks/chain splits), interaction with a bitcoin node (for transaction broadcasting) and general tooling around bitcoin/lightning (transaction parsing, data encoding and decoding, lightning P2P messaging, etc) 
+
+TEOS does not have strong performance requirements,  as data is checked in newly processed blocks approximately every 10 mins. However, if it services a substantial amount of nodes at once or channels with high traffic, it may need to compare processed info against huge amounts of data.
```

Comment:
```suggestion
TEOS does not have strong performance requirements, as data is checked in newly processed blocks approximately every 10 mins. However, if it services a substantial amount of nodes at once or channels with high traffic, it may need to compare processed info against huge amounts of data.
```